### ISSUE TITLE
Ensure that focus remains on the button that was clicked.

### ISSUE THREAD

**Issue by h-parekh** [samvera/hyrax#2988]
### Descriptive summary
Keyboard focus is lost each time a new sort button is clicked.

### Rationale
[WCAG guideline 3.2.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html):

> The intent of this Success Criterion is to ensure that entering data or selecting a form control has predictable effects

### Expected behavior
Focus should remain on the button that was clicked

### Actual behavior
On clicking a button using keyboard, the button loses its focus.
Affected items:

- [ ] Sort By dropdown
- [ ] Results per page dropdown
- [ ] List button
- [ ] Gallery button
- [ ] Masonry button
- [ ] Slideshow button

### Steps to reproduce the behavior

1. Search for something in Nurax - [Example page](https://nurax.curationexperts.com/catalog?locale=en&per_page=20&search_field=all_fields&sort=system_create_dtsi+desc&view=masonry)
2. Tab through using your keyboard to any of the buttons listed above and hit 'Return' key.
3. Notice that the button lost focus

### Related work
https://github.com/samvera/hyrax/issues/1246 is similar `tabbed focus` issue for keyboard users, but may have a different root cause.

**Comment by adamjarling** [comment]
@no-reply @vantuyls Looks like the element `<div id="sortAndPerPage" class="clearfix">` which houses these buttons is coming from Blacklight.  I put the "Blacklight" label on it, and will kick this to the bottom of the current Hyrax WG Sprint Ready column.

**Comment by jcoyne** [comment]
I believe this is happening because activating these buttons is causing a new page to be loaded. So the focus starts over at the top of the page.

**Comment by adamjarling** [comment]
@jcoyne Yes, you're right the selection is causing a new page load which will make retaining focus on the element tricky.   Maybe we evaluate the cost of a solution vs. the cost of keeping this as-is.

**Comment by jcoyne** [comment]
@h-parekh I wonder if the solution for you is to not use Turbolinks?

**Comment by rjkati** [comment]
Issue is still present in Hyrax 3.0.0.pre.rc3.

**Comment by rjkati** [comment]
This issue is still present in Hyrax 4.0.0beta1.

**Comment by scherztc** [comment]
The issue is still present on dev.nurax.samvera.org.

Hyrax Version: 5.0.0.rc2

**Comment by jlhardes** [comment]
Verified this is still present in Hyrax 5.0.0 on https://dev.nurax.samvera.org/. This is also still present on http://demo.projectblacklight.org/ so this is a Blacklight issue that we need to report and share with Blacklight (and possibly contribute a fix).
